jboss-fuse-sap-tool-suite/uitests/tests/org.jboss.tools.fuse.sap.ui.bot.tests/launcher/RedDeer Test Linux x86_64.launch

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is it possible to use a non-platform specific launch configuration?

Reply:
yes, I'm already working on it